shiny/render/_data_frame.py

```diff
@@ -181,7 +185,7 @@ class data_frame(Renderer[DataFrameResult[DataFrameLikeT]]):
     Reactive value of the data frame's edits provided by the user.
     """
 
-    data: reactive.Calc_[DataFrameLikeT]
+    data: reactive.Calc_[IntoDataFrameT]
```

Comment:
The documentation for these isn't quite rendering correctly; they're showing up as attributes, and clicking them doesn't go anywhere.